### PR TITLE
Optimised epaper version

### PR DESCRIPTION
**Pull request by snug**
Updated application name to differentiate version
Removed animations from menu changes
Changed colours to white

This resolves issue #61

### CODE REVIEW

**Comment by jancborchardt** [comment]
This changes the main version though? Isn’t it possible to detect if the screen or device is an e-paper device and automatically adjust?

**Comment by postapczuk** [comment]
Sadly not.

File: LaLauncher/src/main/res/anim/slide_down.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <set xmlns:android="http://schemas.android.com/apk/res/android">
 
     <translate
-            android:duration="400"
```
Comment (postapczuk) [inline LaLauncher/src/main/res/anim/slide_down.xml:5]: Slides are already removed on the epaper branch from the code, so I'suggest to remove this xml completely

File: LaLauncher/src/main/res/anim/slide_up.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <set xmlns:android="http://schemas.android.com/apk/res/android">
 
     <translate
-            android:duration="250"
+            android:duration="0"
```
Comment (postapczuk) [inline LaLauncher/src/main/res/anim/slide_up.xml:5]: Same here, xml to remove

**Review by postapczuk — CHANGES_REQUESTED** [review]

File: LaLauncher/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
-    <color name="colorBackgroundPrimary">#000000</color>
-    <color name="colorBackgroundDimmed">#000000</color>
-    <color name="colorBackgroundFavorite">#66FFFFFF</color>
+    <color name="colorBackgroundPrimary">#FFFFFF</color>
+    <color name="colorBackgroundDimmed">#FFFFFF</color>
+    <color name="colorBackgroundFavorite">#40000000</color>
 
-    <color name="colorTransparent">#00000000</color>
+    <color name="colorTransparent">#00FFFFFF</color>
```
Comment (postapczuk) [inline LaLauncher/src/main/res/values/colors.xml:10]: I don't think this change cause anything as first two digits are "00" which is equal transparency=0. I'd leave that as 00000000

**Comment by postapczuk** [comment]
No one took care to this. Changes were applied on epaper branch.

**Pull request closed without merging**